### PR TITLE
🚀 Release v1.8.0-rc.1

### PR DESCRIPTION
**Pull request by rajankumary2k**
**What this PR does / why we need it**:

- Add release notes for v1.8.0.rc-1

/area release

### CODE REVIEW

File: CHANGELOG/v1.8.0-rc.1.md [hunk 1]
```diff
@@ -0,0 +1,524 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
```
[the remark this answers is not in the document]
Comment (sbueringer) [reply CHANGELOG/v1.8.0-rc.1.md:1]: Please check if the cherry-pick of this PR is also merged: https://github.com/kubernetes-sigs/cluster-api/pull/11021

[the remark this answers is not in the document]
Comment (rajankumary2k) [reply CHANGELOG/v1.8.0-rc.1.md:1]: This Cherry-pick PR is not merged yet.

**Review by chandankumar4 — APPROVED** [review]
lgtm

**Comment by adilGhaffarDev** [comment]
/lgtm 
from my side. @sbueringer is everything looking ok?

**Comment by sbueringer** [comment]
Thx! All good

/lgtm
/approve

**Pull request merged**